server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementIterator.java

```diff
@@ -312,7 +313,8 @@ private boolean shouldReturnDueToLocation(final TabletMetadata tm,
         return true;
       case SUSPENDED:
       case UNASSIGNED:
-        if (shouldBeOnline && (tm.getHostingGoal() == TabletHostingGoal.ALWAYS
+        if (shouldBeOnline && tm.getOperationId() == null && (tm.getHostingGoal()
```

Comment:
Why check the operation id is null here?

Reply:
If the tablet is unassigned, but should be hosted, then we only want to do that if the operationId is null.

Reply:
lol. I missed that it was being checked when computing `shouldBeOnline`.

Reply:
Removed those redundant checks in 2519c98